pkgs/applications/graphics/pizarra/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitLab
+, rustPlatform
+, cargo
+, pkg-config
+, binutils-unwrapped
+, gtk3-x11
+, atk
+, glib
+, librsvg
+, gdk-pixbuf
+, wrapGAppsHook
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "pizarra";
+  version = "1.7.3";
+
+  src = fetchFromGitLab {
+    owner = "categulario";
+    repo = "pizarra-gtk";
+    rev = "v${version}";
+    sha256 = "sha256-XP+P2w6s47JQV4spKeMKe/Ktxid7uokGYH4IEJ5VHSc=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ wrapGAppsHook glib pkg-config gdk-pixbuf ];
```

Comment:
Why is glib and gdk-pixbuf in nativeBuildInputs?

Reply:
As far as I've managed to figure out, `gdk-pixbuf` needs to be present, otherwise I don't get the SVG support for the icons.
I'm not sure if `glib` is needed or not (testing it out right now).

Reply:
`glib` isn't needed for compilation, as it compiles successfully and runs fine.